tox.ini

```diff
@@ -1,23 +1,26 @@
 [tox]
-envlist = 
-# Jython support pending 2.7 support, due 2012-07-15 or so.  See:
-# http://fwierzbicki.blogspot.com/2012/03/adconion-to-fund-jython-27.html
-#    py26,py27,py32,pypy,jython,coverage
+# Jython 2.7rc2 does work, but unfortunately has an issue running
+# with Tox 1.9.2 (apparently its non-interactive console still prints
+# interactive prompts ):
+#   $ tox -e jython
+#   ...
+#   ERROR: InvocationError: Failed to get version_info for jython: could not decode ">>> ... ... ... >>> {'version_info': (2, 7, 0, 'candidate', 2)}\n>>> "
+envlist =
     py26,py27,pypy,py32,py33,py34,pypy3,coverage,docs
 
 [testenv]
-commands = 
+commands =
     python setup.py test -q
 deps = transaction
 
 [testenv:jython]
-commands = 
+commands =
```

Comment:
This whole section should go away -- tox already knows how to substitute `jython` for `python`.